src/components/NavBar/ShoppingCart.tsx

```diff
@@ -1,16 +1,30 @@
 import React from "react";
 import { IconButton, Circle, Stack } from "@chakra-ui/react";
 import { HiOutlineShoppingBag } from "react-icons/hi";
+import { switchCartDisplay } from "@/store/reducer/cartControlSlice";
+import { useDispatch } from "react-redux";
 
 interface Props {
   quantity: number;
   loginState: boolean;
 }
 
 const ShoppingCart = ({ quantity, loginState }: Props) => {
```

Comment:
The name of this file is as same as another folder in the component. Having 2 same name files or folders is bad practice as IDE or team members may not pick up the correct one they need.

Maybe change this one to something like "ShoppingCartButton" or you name it.

Reply:
> The name of this file is as same as another folder in the component. Having 2 same name files or folders is bad practice as IDE or team members may not pick up the correct one they need.
> 
> Maybe change this one to something like "ShoppingCartButton" or you name it.

/NavBar/ShoppingCart.tsx was renamed as /NavBar/ShoppingCartButton.tsx.